docs/4.x/archiving.md

```diff
@@ -41,21 +41,24 @@ While pre-computing archive data seems of course superfluous for 1000 page views
 
 ## When?
 
-By default, archive data is calculated and cached **on-demand**. When a specific report is requested, Piwik will check if the required archive data exist and generate it if not.
+### On-demand Archiving
+
+By default, archive data is calculated and cached **on-demand**. When a specific report is requested, Piwik will check if the required archive data exists and generate it if it does not.
```

Comment:
Maybe we also add something like this:

If the report was archived recently (within configured amount of seconds in time_before_today_archive_considered_outdated) then no new archive is generated.

Reply:
👍